tokens/lift.json

```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x513C3200F227ebB62e3B3d00B7a83779643a71CF",
+    "aurora_address": "0x513C3200F227ebB62e3B3d00B7a83779643a71CF",
```

Comment:
This is not an address of the bridged token. If you want to have an address of the bridged token - please deploy it using https://rainbowbridge.app/deploy